cli/cli/helpers/engine_manager/engine_manager.go

```diff
@@ -33,6 +33,7 @@ const (
 	waitUntilEngineStoppedTries    = 5
 	waitUntilEngineStoppedCoolOff  = 5 * time.Second
 	defaultHttpLogsDatabasePortNum = uint16(9714)
+	defaultLogAggregatorPortNum    = uint16(9714)
```

Comment:
Why do we have both `defaultHttpLogsDatabasePortNum` and `defaultLogAggregatorPortNum`?

Reply:
removed